physical/couchdb/couchdb.go

```diff
@@ -227,6 +242,10 @@ func (m *CouchDBBackend) Delete(key string) error {
 func (m *CouchDBBackend) List(prefix string) ([]string, error) {
 	defer metrics.MeasureSince([]string{"couchdb", "list"}, time.Now())
 
+	if m.prefixed {
+		prefix = "$" + prefix
```

Comment:
Can we use another prefix other than "$"? If there's a tool that supports storage migration, for instance, some underlying storages might not support having symbols as part of the path (though I can't think any off the top of my head) and make migration/data manipulation non-trivial.

Reply:
Other storages wouldn't need this in the path. Note that the path is translated internally; the entry.Key is not updated. By the time the entry gets back from the underlying layer, the prefix is already stripped off.

Reply:
Ahh, I see what you mean. What's the ID used for in here: https://github.com/hashicorp/vault/pull/3479/files#diff-2bbba2fac19e5e53717543473f9a28efR318, is that something internal/specific to CouchDB?

Reply:
Yeah, it's a document store so I think that's the ID of the document.